src/dom/RequestAnimationFrame.js

```diff
@@ -116,7 +126,7 @@ var RequestAnimationFrame = new Class({
         {
             var d = Date.now();
 
-            var delay = Math.max(16 + _this.lastTime - d, 0);
+            var delay = Math.min(Math.max(_this.target * 2 + _this.tick - d, 0),_this.target);
```

Comment:
Unless `target` is given a default value > 0, there will never be a delay set, because this will always return zero.
Also, space after commas in argument lists.

Reply:
I have fixed the comma

the `target` is set from `start` method always - i can set so sane default for example 16 (that will be 60 fps) but that will anyway be overwritten in `start` method - what do you think?